tests/FixerFactoryTest.php

```diff
@@ -250,6 +251,82 @@ public function testNoDoubleConflictReporting()
         );
     }
 
+    public function testSetWhitespacesConfig()
+    {
+        $factory = new FixerFactory();
+        $config = new WhitespacesFixerConfig();
+
+        $fixer = $this->prophesize('PhpCsFixer\Fixer\WhitespacesAwareFixerInterface');
+        $fixer->getName()->willReturn('foo');
```

Comment:
This method isn't called so you don't need to define it.

Reply:
If I don't define it I get

```
1) PhpCsFixer\Tests\FixerFactoryTest::testSetWhitespacesConfig
Prophecy\Exception\Call\UnexpectedCallException: Method call:
  - getName()
on Double\WhitespacesAwareFixerInterface\P19 was not expected, expected calls were:
  - setWhitespacesConfig(exact(PhpCsFixer\WhitespacesFixerConfig:00000000010848210000000006cade73 Object (
    'indent' => '    '
    'lineEnding' => '
'
)))
```

Reply:
Ah yes, I read the code too fast.